plasmapy/mathematics/tests/test_dispersion.py

```diff
@@ -35,7 +35,7 @@ def test_plasma_dispersion_func(w, expected):
 
     Z_of_w = plasma_dispersion_func(w)
 
-    assert np.isclose(Z_of_w, expected, atol=1e-12 * (1 + 1j), rtol=1e-12), \
+    assert np.isclose(Z_of_w, expected, atol=1e-12 * (1 + 1j), rtol=1e-6), \
```

Comment:
I was pretty much guessing blindly with the 1e-6 value. Could you try something lower, closer to the old 1e-12? Go as low as you can - ideally we'd like to both be accurate in testing the old tabularized values and not so accurate that machine precision will break these tests.

Reply:
Okay! I'll work on this.